service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderControllerTest.java

```diff
@@ -81,6 +81,8 @@
     private static final String LOCAL_AUTHORITY_CODE = "example";
     private static final String FAMILY_MAN_CASE_NUMBER = "SACCCCCCCC5676576567";
     private static final String CASE_ID = "12345";
+    private final byte[] pdf = {1, 2, 3, 4, 5};
+    private Document document;
```

Comment:
maybe not relevant to this PR - but I think I can see a case for a DocmosisBaseTest class we can extend upon. Do you think this is a good idea? I can add to tech debt

Reply:
Yes this could be a tech debt ticket :) agreed not for this pr